Foundation/NSError.swift

```diff
@@ -481,7 +498,7 @@ public protocol _BridgedNSError : __BridgedNSError, RawRepresentable, _Objective
 
 /// Describes a bridged error that stores the underlying NSError, so
 /// it can be queried.
-public protocol _BridgedStoredNSError : __BridgedNSError, _ObjectiveCBridgeableError, CustomNSError, Hashable {
+public protocol _BridgedStoredNSError : __BridgedNSError, _ObjectiveCBridgeableError, CustomNSError, Hashable, CustomStringConvertible {
```

Comment:
Mind moving the `CustomStringConvertible` to the extension below?

Reply:
@compnerd That is what I initially attempted, but "protocol extensions cannot have inheritance clauses"

Reply:
A conditional conformance would not work either, because at some point each concrete type (`POSIXError`, `CocoaError`, `URLError`) would need to declare `CustomStringConvertible` conformance.

Reply:
Hmm, I wonder if that is something special here, because I've done extensions for conformances previously.